docs/guides/fragments/flutter/flutterWarning.md

```diff
@@ -0,0 +1,8 @@
+
+<amplify-callout warning>
+
+### The API Category is not yet implemented in Flutter Preview.
+
+#### Please ignore the Flutter Guides section for now. 
```

Comment:
I would suggest some other than 'Please ignore...'.  Maybe something like "This section will be updated once the category becomes available."